TheOracle2/Interactions/SlashCommands/RollCommandGroup.cs

```diff
@@ -7,9 +7,10 @@
 
 namespace TheOracle2;
 
-public class PlayerRollCommand : InteractionModuleBase
+[Group("roll", "Make an action roll (p. 28) or progress roll (p. 39). For oracle tables, use '/oracle'")]
```

Comment:
i figure `/roll` is what people will try first for action/progress rolls, and this means people only have to remember one command to execute the most common game functions. 

i think a command group is the best solution here. for one, `/roll` is very brief. for another, as top-level commands they'd have to be named `/roll-action` and `/roll-progress` to function that way, which doesn't actually save many (if any) keystrokes